packages/react/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { BUTTON_GROUP } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * renders the buttons
+ *
+ * @param {object} props props object
+ * @param {Array.<{copy: string}>} props.buttons array of buttons
+ * @param {string} props.buttons[].copy Button copy
+ * @param {string} props.buttons[].link URL for the button item
+ * @param {object} props.buttons[].renderIcon Optional icon type
+ * @returns {*} button components
+ */
+const ButtonGroup = ({ buttons }) => {
+  return featureFlag(
+    BUTTON_GROUP,
+    <div
+      className={`${prefix}--buttongroup`}
+      data-autoid={`${stablePrefix}--button-group`}>
+      {buttons.map((button, key) => {
+        return (
+          <Button
+            key={key}
+            data-autoid={`${stablePrefix}--button-group-${key}`}
+            renderIcon={button.renderIcon || null}
+            href={button.link}
```

Comment:
I'm looking at the `Button` itself in carbon, and don't see it outwardly supporting onClick. I think however there is an `other` spread operator that gets passed in, which I'm assuming would allow for other props like onClick get passed in. While I'm debating if we should explicitly provide a prop for onClick, we should however think about if we want to support the other built-in props for `Button`, as well as also just passing `others` props through to the Button itself.

Reply:
Yeah, I think spread would allow for onClick and more. I was mistaken. It looks like Carbon's button is handling the validation of the `link`/`href` (which could better align here) so if you set `link` to undefined/null, or just don't provide it it'll turn into a button instead of an anchor which is what I was asking about here.

Reply:
@photodow updated to allow in a spread to go through for each `Button`, also for the `link`. This should accommodate for any of the additional `Button` attributes that the user can pass through for each object in the array.